doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -441,6 +441,109 @@ Here are examples:
    end
    where "n + m" := (plus n m).
 
+Enabling and disabling notations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. cmd:: {| Enable | Disable } Notation {? {| @string | @qualid } } {? := @one_term } {? ( {+, @toggle_notation_flag } ) } {? {| : no scope | : @ident } }
+   :name: Disable Notation
+
+   .. insertprodn toggle_notation_flag toggle_notation_flag toggle_notation_flag
+
+   .. prodn::
+      toggle_notation_flag ::= all
+      | only parsing
+      | only printing
+      | in custom @ident
+      | in constr
+
+   Enables or disables notations previously defined with
+   :cmd:`Notation` or :cmd:`Notation (abbreviation)`. At least one of
+   :token:`string`, :token:`qualid`, :token:`one_term` or scope must be
+   provided. If more than one is provided, the notations enabled or
+   disabled are the ones meeting the conjunction of these constraints.
+
+   This command supports the :attr:`local` and :attr:`global`
+   attributes.
+
+   .. note::
+      This command has no effect on parser modifications made with
+      :cmd:`Reserved Notation`.
+
+   :n:`@string`
+      Notations to enable or disable. :n:`@string` can be a single
+      token in the notation such as "`->`" or a pattern that matches
+      the notation. See :ref:`locating-notations`.
+
+   :n:`@qualid`
+      :ref:`Abbreviation <Abbreviations>` to enable or disable.
+
+   :n:`{? := @one_term }`
+      Enable or disable notations involving :token:`one_term`.
+
+   :n:`{? {| : no scope | : @ident } }`
```

Comment:
Helloooo out there, you skipped this one again.  Please do it or say why not.

Reply:
Sorry, I was sure I answered but obviously not. As I said above https://github.com/coq/coq/pull/12324#discussion_r1009212477, I'm unfortunately unable to write the appropriate thing in `common_edit.mlg`. But you're perfectly right this should be `scope_name`.

Here is what I tried
```
| REPLACE [ "Enable" | "Disable" ] "Notation" OPT [ string | qualid ] OPT ( ":=" one_term ) OPT ( "(" LIST1 enable_notation_flag SEP "," ")" ) OPT [ ":" ident | ":" "no" "scope" ]
| WITH    [ "Enable" | "Disable" ] "Notation" OPT [ string | qualid ] OPT ( ":=" one_term ) OPT ( "(" LIST1 enable_notation_flag SEP "," ")" ) OPT [ ":" scope_name | ":" "no" "scope" ]
```
(copied from `orderedGrammar`)
but I get the following error
```
make doc_gram_rsts
Error: Undefined nonterminal `one_term` in edit: [ "Enable" | "Disable" ] "Notation" OPT [ string | qualid ] OPT ( ":=" one_term ) OPT ( "(" LIST1 enable_notation_flag SEP "," ")" ) OPT\
 [ ":" scope_name | ":" "no" "scope" ]
Error: Can't find '[ "Enable" | "Disable" ] "Notation" OPT [ string | qualid ] OPT ( ":=" one_term ) OPT ( "(" LIST1 enable_notation_flag SEP "," ")" ) OPT [ ":" ident | ":" "no" "scope\
" ]' in edit for 'syntax'
2 error(s)
```
Any help here would be welcome.

Reply:
Easily solved.  `constr` is renamed to `one_term` towards the end of the file, after your edit.  If you temporarily insert `| PRINT` at the point of your failing edit, `doc_grammar` will print the current definition of the containing nonterminal (perhaps `simple_tactic`).  Probably just use `constr` here instead of `one_term` in both lines.

Reply:
Thanks a lot, that worked like a charm!